<?xml version="1.0"?>

<launch>
<node pkg = "osrf_gear" name = "gear_sim" type = "gear.py" args = "-f $(find osrf_gear)/config/final04-2.yaml $(find ariac_comp)/config/Config_Comp.yaml"/>

</launch>
